bin/hdfcoinc/pycbc_combine_coincident_events

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+"""
+This program combines together a set of STATMAP files from disjoint times.
+The resulting file would contain triggers from the full set of input files
+"""
+import numpy
+import argparse
+import logging
+import h5py
+
+import pycbc
+import pycbc.version
+
+def com(f, files, group):
+    """ Combine the same column from multiple files into another file f"""
+    f[group] = numpy.concatenate([fi[group][:] if group in fi else numpy.array([], dtype=numpy.uint32) for fi in files])
+
+def com_with_detector_key(f, files, group):
+    """
+    Combine data from multiple files where the group is dependent on which of
+    the two ifos the data belongs to, e.g. time1, time2 or trigger_id1,
+    trigger_id2. This function checks which group should be used
+    for each file, as each file might have a different convention, and combines
+    according to the attributes set in f.
+    """
+    # We have to be careful here, because in these cases the group, for
+    # example time1 or time2 encodes a detector name, stored in the attributes.
+    # It may not be the *same* detector for each file.
+
+    # What detector are we dealing with
+    if group.endswith('1'):
+        ifo_name = f.attrs['detector_1']
+    elif group.endswith('2'):
+        ifo_name = f.attrs['detector_2']
+    else:
+        raise ValueError("Input must end in 1 or 2, got %s" % inp_str)
```

Comment:
basic method looks fine, but prefer to say 'group name must end in 1 or 2' in the error msg - and what's 'inp_str'?